hc/api/models.py

```diff
@@ -87,6 +89,8 @@ def get_status(self):
         now = timezone.now()
 
         if self.last_ping + self.timeout + self.grace > now:
+            if self.status == "often":
```

Comment:
This doesn't need to be nested. We really don't require to check this condition`self.last_ping + self.timeout + self.grace > now` to return a status for _often_ checks